src/components/SampleList/SampleList.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useMemo } from "react";
+import {
+  IonButton,
+  IonChip,
+  IonCol,
+  IonGrid,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonListHeader,
+  IonNote,
+  IonRow,
+  IonSearchbar,
+  SearchbarInputEventDetail,
+} from "@ionic/react";
+import { paths } from "../../Router";
+import { search as searchIcon } from "ionicons/icons";
+import NoneOr from "../NoneOr/NoneOr";
+import { SubmissionMetadata } from "../../api";
+import { IonSearchbarCustomEvent } from "@ionic/core/dist/types/components";
+import { useMiniSearch } from "react-minisearch";
+import { getSubmissionSamples } from "../../utils";
+
+interface SampleListProps {
+  submission: SubmissionMetadata;
+  collapsedSize?: number;
+}
+
+const SampleList: React.FC<SampleListProps> = ({
+  submission,
+  collapsedSize = 5,
+}) => {
+  const searchElement = React.useRef<HTMLIonSearchbarElement>(null);
+
+  const [collapsed, setCollapsed] = React.useState(true);
+
+  const [showSearch, setShowSearch] = React.useState(false);
+  const [searchTerm, setSearchTerm] = React.useState<string | null>();
+
+  const samples = useMemo(
+    // This is reversed so that the most recent samples are shown first
+    () => getSubmissionSamples(submission).toReversed(),
+    [submission],
+  );
+
+  const { search, searchResults } = useMiniSearch(samples, {
+    fields: ["samp_name"],
+    idField: "_index",
+    searchOptions: {
+      prefix: true,
+      combineWith: "AND",
+    },
+  });
+
+  const handleSearchInput = (
+    event: IonSearchbarCustomEvent<SearchbarInputEventDetail>,
+  ) => {
+    setSearchTerm(event.target.value);
+    if (event.detail.value) {
+      search(event.detail.value);
+    }
+  };
+
+  const handleSearchCancel = () => {
+    setSearchTerm(null);
+    setShowSearch(false);
+  };
+
+  const displaySamples = searchTerm ? searchResults || [] : samples;
+
+  return (
+    <>
+      <IonListHeader>
+        <IonLabel>Samples {samples && <>({samples.length})</>}</IonLabel>
+        <IonButton routerLink={paths.sampleCreate(submission.id)}>
+          New
+        </IonButton>
+      </IonListHeader>
+
+      <IonGrid className={showSearch ? "ion-hide" : ""}>
+        <IonRow class="ion-justify-content-between">
+          <IonCol size="auto">
+            <IonChip outline>
+              Template: {submission.metadata_submission.templates[0]}
```

Comment:
According to the types, it can be an empty list.

https://github.com/microbiomedata/nmdc-field-notes/blob/c98d6ed89ce1c9b24983efbb4acfbd3fa8135593/src/api.ts#L29

Do you still want to show the chip in that situation?